benefits/static/css/styles.css

```diff
@@ -174,6 +174,10 @@ footer .footer-links a {
   background: #212121;
 }
 
+.home label {
```

Comment:
I would change the `margin-top: 20vh` to be applied to the outer parent `.buttons` class instead of adding it to `label`. Also, it's a good practice generally to prefer adding spacing styles to classes (not to all HTML elements). 
<img width="493" alt="image" src="https://user-images.githubusercontent.com/3673236/163876996-24718881-6031-4772-9ca3-c48b6554ce1d.png">

Otherwise, this pr looks great!

Reply:
Thank you for this feedback and the tip on good practices! 🎨 

I agree this makes more sense:
![image](https://user-images.githubusercontent.com/25497886/163880145-742d5928-b391-4e33-940f-57503a590b81.png)